salmon_occurrence/salmon_db.py

```diff
@@ -0,0 +1,77 @@
+from sqlalchemy import Column, Integer, String, Enum, Boolean, Float, ForeignKey, Index, UniqueConstraint
+from sqlalchemy.ext.declarative import declarative_base
+from geoalchemy2 import Geometry
+
+Base = declarative_base()
+
+class Region(Base):
+    __tablename__="region"
+    __table_args__ = {'schema': 'salmon_geometry'}
```

Comment:
Extract the schema name to a single declaration, as you have in the migration code? Possibly it would be a good thing to make it common between them.